packages/schema/src/dialects/mysql.ts

```diff
@@ -54,6 +54,7 @@ export function rawColumnToColumn(rawColumn: RawColumn): Column {
 		is_generated: !!rawColumn.EXTRA?.endsWith('GENERATED'),
 		is_nullable: rawColumn.IS_NULLABLE === 'YES',
 		is_unique: rawColumn.COLUMN_KEY === 'UNI',
+		is_indexed: rawColumn.COLUMN_KEY === 'MUL',
```

Comment:
`MUL` is for non unique index(including composite), I see directus [doesnt support composite indexes](https://github.com/directus/directus/issues/19377) today as well, so i went ahead with this, another option is to use `SEQ_IN_INDEX` from `INFORMATION_SCHEMA.STATISTICS` , which is not straight forward either